test/runtests.jl

```diff
@@ -1,5 +1,8 @@
 using MathOptInterface, JuMP, Dualization, Test
 
+# Solvers
+using GLPK, CSDP, COSMO, ECOS
```

Comment:
It's better to have this in a script in `Solvers` so that it's easy to skip it if they are not installed